slack_sdk/audit_logs/v1/client.py

```diff
@@ -190,7 +199,7 @@ def api_call(
         return self._perform_http_request(
             http_verb=http_verb,
             url=url,
-            body_params=body_params,
+            body=body_params,
```

Comment:
As `_perform_http_request` is an internal method, we can safely rename this arg name